### PR TITLE
Update ideas page documentation

### PR DESCRIPTION
**Pull request by adelq**
I found a few outdated remarks on the ideas page of the documentation, there could be more regarding the current implementation of linked adapters but my current knowledge of the parameter revamp is quite limited

### CODE REVIEW

File: doc/ideas.rst [hunk 1]
```diff
@@ -43,7 +43,7 @@ to add new adapter types in the feature.
     anywhere,``-b ADAPTER``, ``-a ...ADAPTER...`` ???
     unconditional,``-u +10``,``-a 10...`` (collides with colorspace)
     unconditional,``-u -10``,``-a ...10$``
-    linked,(not implemented),``-a ADAPTER...ADAPTER`` or ``-a ^ADAPTER...ADAPTER``
+    linked,``-a ADAPTER...ADAPTER``,``-a ADAPTER...ADAPTER`` or ``-a ^ADAPTER...ADAPTER``
```
Comment (marcelm) [inline doc/ideas.rst]: This line now mentions `-a ADAPTER...ADAPTER` twice

**Comment by marcelm** [comment]
Thanks. Please ignore the Travis failure; it does not have anything to do with this change.

File: doc/ideas.rst (hunk 1 above)
Comment (adelq) [inline doc/ideas.rst]: Presumably because the existing documented parameters for linked adapters match up with the intended future specification?

Comment (marcelm) [inline doc/ideas.rst]: Sorry, did not realize it was a table. Of course.

**Pull request merged**